packages/dicomImageLoader/src/imageLoader/wadors/metaData/USHelpers.ts

```diff
@@ -0,0 +1,67 @@
+import getNumberValues from './getNumberValues';
+import getSequenceItems from './getSequenceItems';
+
+function getUSEnhancedRegions(metadata) {
```

Comment:
Docs please

Reply:
Also, this would be a great place to have a jest unit test, as there is just enough complexity to warrant a unit test, but not quite enough for have a full set of integration tests on this.